### DISCUSSION TITLE
How to run the test cases in linux platform using webdriverio.

### DISCUSSION THREAD

**Discussion by madhuUYO** [webdriverio/webdriverio#8026] [Q&A]
I am new to webdriverIO, earlier I was using protractor and I need to migrate using webdriverIO. Does webdriverIO supports cross platform testing? Can I execute my test case in linux also.

**Comment by WillBrock** [comment]
Yes, you can run on linux, mac, windows.

**Reply by madhuUYO** [reply]
Now I am using ****selenium-stand alone service**** in my conf file with which I can automate in three browsers(firefox, edge and chrome). But for Mac what extra settings, I need to do and what is the protocol I need to use. Please share config file.

**Reply by christian-bromann** [reply]
> But for Mac what extra settings

No extra settings needed. You just have to have the browser installed on your machine.
